packages/parse5/lib/parser/index.test.ts

```diff
@@ -110,4 +111,41 @@ describe('parser', () => {
             expect(onItemPop).toHaveBeenLastCalledWith(bodyElement.childNodes[0], bodyElement);
         });
     });
+
+    describe('rawtext parsing', () => {
+        it.each([
+            ['iframe'],
+            ['noembed'],
+            ['noframes'],
+            ['noscript'],
+            ['script'],
+            ['style'],
+            ['textarea'],
+            ['title'],
+            ['xmp'],
```

Comment:
I tried to find all the "raw text" tags I knew of, based on [the Firefox source code](https://searchfox.org/mozilla-central/rev/72d959d715d3f832328057600811c09ee5195ae1/parser/html/nsHtml5Tokenizer.cpp#54-87).

Again, all these tests were already passing except for `noframes`.